src/components/ui/follow/index.tsx

```
@@ -2,33 +2,80 @@ import React, { useState } from 'react';
+    userId: string;
 }
 
-export default function Follow({ onClose }: FollowProps) {
-    const [search, setSearch] = useState('');
-    const filtered = mockFollowers.filter(f =>
-        f.username.includes(search) || f.name.includes(search)
-    );
+export default function Follow({ onClose, userId }: FollowProps) {
+    // const [search, setSearch] = useState('');
+
+    const { data: followingData, refetch: refetchFollowing } = useQuery(ProfileGQL.QUERIES.GET_FOLLOWING, {
```

ESLint

[error] 'refetchFollowing' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)